chapters/data-collection.tex

```diff
@@ -1,362 +1,286 @@
+
+\subsection{Secure data storage}
+\textbf{Encryption at rest} is the only way to ensure that PII data remains private when it is stored on someone else's server on the open internet. You must keep your data encrypted on the server whenever PII data is collected.
+Encryption makes data files completely unusable without access to a security key specific to that data -- a higher level of security than password-protection.
+Encryption at rest requires active participation from the user; and you should be fully aware that if your private key is lost, there is absolutely no way to recover your data.
+
+You should not assume that your data is encrypted by default: indeed, for most CAPI software platforms, encryption needs to be enabled by the user.
+To enable it, you must confirm you know how to operate the encryption system and understand the consequences if basic protocols are not followed.
+When you enable encryption, the service will allow you to download -- once -- the keyfile pair needed to decrypt the data.
+You must download and store this in a secure location, such as a password manager. Make sure you store keyfiles with descriptive names to match the survey to which they correspond.
+Any time anyone accesses the data - either when viewing it in the browser or downloading it to your computer - they will be asked to provide the keyfile.
+Only project teams members names in the IRB are allowed access to the private keyfile.
+
+To proceed with data analysis, you typically need a working copy of the data accessible from a personal computer. The following workflow allows you to receive data from the server and store it securely, without compromising data security.
+
+\begin{enumerate}
+	\item Download data
+	\item Store a ``master'' copy of the data into an encrypted location that will remain accessible on disk and be regularly backed up
+	\item Secure a ``gold master'' copy of the raw data in a secure location, such as a long-term cloud storage service or an encrypted physical hard drive stored in a separate location. If you remain lucky, you will never have to access this copy -- you just want to know it is out there, safe, if you need it.
+
+\end{enumerate}
+
+This handling satisfies the rule of three: there are two on-site copies of the data and one off-site copy, so the data can never be lost in case of hardware failure.
+In addition, you should ensure that all teams take basic precautions to ensure the security of data, as most problems are due to human error.
+Ideally, the machine hard drives themselves should also be encrypted, as well as any external hard drives or flash drives used.
+All files sent to the field containing PII data, such as sampling lists, should at least be password protected. This can be done using a zip-file creator.
```

Comment:
Is our recommendation this or does it need to be encrypted?

Reply:
If you mean files sent to field, I think they should be encrypted.

Reply:
Do we have instructions for this? I don't what the best practice is for encrypting say spreadsheets

Reply:
I think that it should be encrypted and veracrypt is appropriate to encrypt an Excel file and then send it over email

Reply:
This is covered in the [write up](https://paper.dropbox.com/doc/Data-Encryption--AtAanw31w3IzTGCtWyIAJAJOAg-2n7oTb8aO8VT9oP2ukA2i) I did today. This will be included in the wiki article on encryption. So I think we can simply say that files with PII data sent to the file should be encrypted, and then have a side note to the wiki page for encryption if we have not already had that in this chapter yet

Reply:
The data security standard is a good place to link to, but it is not public yet

Reply:
issue #344